src/OSPSuite.R/Services/DataImporterTask.cs

```diff
@@ -147,5 +185,106 @@ public void SaveConfiguration(ImporterConfiguration configuration, string path)
             serializer.Serialize(configuration, serializationContext).Save(path);
          }
       }
+
+      /// <summary>
+      /// Add an error column to the configuration if no error column is present.
+      /// If the configuration already has an error column, the mehtod does nothing.
+      /// </summary>
+      /// <param name="configuration">Configuration object</param>
+      public void AddError(ImporterConfiguration configuration)
+      {
+         //Add a new error column only of the configuration does not have an error column yet
+         var errorParameter = GetError(configuration);
+         if (errorParameter == null)
+         {
+            Column errorColumn = new Column()
+            {
+               Name = _columnInfos.First(ci => ci.IsAuxiliary()).DisplayName,
+               Dimension = _dimensionFactory.Dimension("Concentration (molar)"),
+               Unit = new UnitDescription("µmol/l"),
+               ErrorStdDev = "Arithmetic Standard Deviation"
+            };
+            configuration.AddParameter(new MappingDataFormatParameter("Error", errorColumn));
+         }
+      }
+
+      public void AddGroupingColumn(ImporterConfiguration configuration, string columnName)
+      {
+         if (configuration.Parameters.Any(p => p.ColumnName == columnName))
+            return;
+
+         DataFormatParameter parameter = new GroupByDataFormatParameter(columnName);
+         configuration.AddParameter(parameter);
+      }
+
+      public void SetAllLoadedSheet(ImporterConfiguration configuration, string[] sheets)
+      {
+         configuration.ClearLoadedSheets();
+         sheets.Each(sheet => configuration.AddToLoadedSheets(sheet));
+      }
+
+      public void SetAllLoadedSheet(ImporterConfiguration configuration, string sheets)
+      {
+         configuration.ClearLoadedSheets();
+         configuration.AddToLoadedSheets(sheets);
+      }
+
+      public string[] GetAllGroupingColumns(ImporterConfiguration configuration)
+      {
+         return configuration.Parameters.Where(p => (p is MetaDataFormatParameter) || (p is GroupByDataFormatParameter)).Select(p => p.ColumnName).ToArray();
```

Comment:
proably worth putting in two lines by splitting a .Select. I need to scroll left and right to review the code.